src/store/types/SettingsKeys.d.ts

```diff
@@ -0,0 +1,9 @@
+export interface SettingsKeys {
+  authorEmail: String;
+  athorName: String;
+  projectsFolder: String;
+  hasAi2Html: Boolean;
+  projects: String[];
+}
+
+export type Key = keyof SettingsKeys;
```

Comment:
Wait why do we need this?

Reply:
We don't need it! It's just a handy convenience for VSCode so any time you do `getStoreValue` or `updateStoreValue` it'll give auto-complete suggestions in case I forget what they are or don't feel like looking it up. I'll adjust it a bit so it doesn't look so critical.

Reply:
Gotcha, I'm fine with that then, just making sure you're not subtly pushing us toward adopting typesscript 😅

Reply:
Nah I'm just a huge fan of IntelliSense + JSDoc... which may or may not be Microsoft trying to get people to adopt typescript 😰 ! I've got a ton of thoughts on the matter, but I know from my experience with Keystone and other "modern" frameworks and the work they do in Typescript translates to really valuable "in-the-editor" documentation via ItelliSense that has been priceless for me. And JSDoc can serve the serve the same purpose without fully committing to TS while still getting the benefits of documented "in-the-editor" code.